addon/styles/addon.scss

```diff
@@ -18,113 +18,113 @@ $modal-bow-shadow: rgba(0, 0, 0, .25);
       padding: 0;
       background-color: $frost-color-white;
       box-shadow: 0 3px 8px $modal-bow-shadow;
+    }
+  }
 
-      .frost-button {
-        outline: none;
-        background: transparent;
```

Comment:
This diff looks ugly, but I basically just pulled the majority of the styles out from a couple nesting levels to live just below the `.frost-about-dialog` selector.